pkg/lib/localstore/filestore/helpers.go

```diff
@@ -0,0 +1,38 @@
+package filestore
+
+import (
+	"github.com/anyproto/anytype-heart/pkg/lib/pb/storage"
```

Comment:
Some files include imports in incorrect order

Reply:
yea, thanks, accidentally turned off GCI hook